python/tank/bootstrap/manager.py

```diff
@@ -64,15 +64,13 @@ def __repr__(self):
     def _get_pipeline_configuration(self):
         """
         The pipeline configuration that is being operated on.
-        By default, the primary pipeline config will be used.
+        By default, the manager will attempt to automatically determine
```

Comment:
Minor: These new docstring lines feels oddly placed. Seems like they'd be more at home in the class docstring.